descriptors/ModuleDescriptor-template.json

```diff
@@ -66,7 +66,7 @@
     },
     {
       "id": "instance-storage",
-      "version": "7.1",
+      "version": "8.0",
```

Comment:
Why is this a breaking compatibility change?

Reply:
Probably yes

Reply:
> Probably yes

Yes, to causing confusion?

How could I have described that differently? Are you aware of the difference in FOLIO between interface and implementation compatibility?

Reply:
Sorry Marc I didn't catch it at once. I will fix it. It should be 7.2 for the instance-storage and 0.3 for the instance-storage-batch

Reply:
Why does the interface version need to change in any way? As far as I can tell, the interface hasn't been changed in any way.

What has changed is that the implementation is choosing to reject instances where the resource type reference does not refer to a resource type reference record?

Reply:
Is it fine now? If no please provide your input how it should look like